Vision/owlbot.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Google LLC
+# Copyright 2021 Google LLC
```

Comment:
Maybe keep the original year? We changed the filename, but majority stays the same.